.github/pull_request_template.md

```diff
@@ -1,19 +1,25 @@
-## Done
+### Done
 
-[List of work items including drive-bys]
+- [Add clear and detailed changes description]
 
-## Issue / Card
+---
 
-Fixes #
-
-## QA
+### How to QA
 
-- Pull the branch
-- Run the site using the dotrun snap with `$ dotrun`
-- View the site locally in your web browser at: http://0.0.0.0:8004/
+- Visit the site in your browser at: [Replace with demo link]
```

Comment:
I don't think we should remove completely the dotrun commands references and local development references. 
Demos are only available for people in the organisation. We should find a way to keep what we had but also suggest to add demo link in the PR description for us.

Reply:
There will be a link to https://github.com/canonical-web-and-design/snapcraft.io/blob/master/README.md#local-development where you can find all the info about the local development. Do you think that's not enough?

Reply:
well when reviewing a PR I don't want to be clicking on some random link that sends me away of the PR. I strongly think that having this link already there is important. 

The strength of dotrun is that it a unique command to run. I believe that having it with the link in the PR template is not noise but it is actually helpful.

I think the draft exercise is good thing but not a pattern and we should keep a local first QA and let the developer then replace the links with demo if she wants.

Reply:
updated